tests/test_schema.py

```diff
@@ -0,0 +1,221 @@
+"""
+
+Tests for schema and examples.
+
+"""
+
+import json
+import glob
+from pathlib import Path, PurePath
+import datetime
+
+import pytest
+import yaml
+
+import jsonschema
+
+from copy import deepcopy
+
+
+TESTDIR = Path(__file__).parent.absolute()
+
+
+def test_schema_basic_json_syntax():
+    """Confirm that schemas are valid JSON"""
+
+    # find and parse all schema files
+    path = PurePath(TESTDIR, "../schema/definitions/*/schema/*.json")
+    schema_file_paths = glob.glob(str(path))
+
+    # check that schemas are there
+    assert len(schema_file_paths) > 0
+
+    for schema_file_path in schema_file_paths:
+        _parse_json(schema_file_path)
+
+
+def test_schema_example_filenames():
+    """Assert that all examples are .yml, not .yaml"""
+
+    searchpath = str(PurePath(TESTDIR, "../schema/definitions/*/examples/*.*"))
+    filenames = glob.glob(searchpath)
+
```

Comment:
When you use pathlib, rather use the `pathlib.Path().glob()` function

Reply:
✔